style-guide.md

```diff
@@ -129,15 +129,15 @@ Depending on the size of the company, the team description could also be the com
 
 DO: If it is product development a link to find out more about the product would be helpful if it's available.
 
-"The candidate will be working on our product development team on our [Analytics Product](http://google.com)"
+"The candidate will be working on our product development team on our [Analytics Product](http://google.com)" <!-- Possible NDA and/or company discretion conflicts from linking the product -->
 
 Do: mention the development methodology/culture if that's important to you in selecting a candidate
 
 
 <!--
 improve this
 -->
-"The candidate will be working on our product development team, in a fast paced environment, we use Scrum"
+"The candidate will be working on our product development team, in a fast paced environment, we use Scrum.  We usually expect an arbitrary contribution before making an offer."
```

Comment:
*IF* there's a "try before we buy" process, that would be good to know. However, that is almost certainly the kind of thing that will eliminate those currently employed from consideration. For someone *unemployed*, this can be awesome _as long as it is compensated_. If you're trying to indicate that prospective employees will be initially offered a two-week consulting contract, just say *that* rather than "arbitrary contribution."